manager/src/main/java/io/gingersnapproject/micrometer/CacheManagerMetricsProducer.java

```diff
@@ -0,0 +1,32 @@
+package io.gingersnapproject.micrometer;
```

Comment:
Should this be nested within `io.gingersnapproject.metrics`?

Reply:
sure!